src/main/java/mif/vu/ikeea/Manager/TeamCreationManager.java

```diff
@@ -0,0 +1,27 @@
+package mif.vu.ikeea.Manager;
+
+import mif.vu.ikeea.Entity.Repository.TeamRepository;
+import mif.vu.ikeea.Entity.Repository.UserRepository;
+import mif.vu.ikeea.Entity.Team;
+import mif.vu.ikeea.Factory.TeamFactory;
+import mif.vu.ikeea.Service.TeamService;
+import mif.vu.ikeea.Service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.Optional;
+
+public class TeamCreationManager {
+
+    private TeamService teamService;
+
+    public TeamCreationManager(TeamService teamService){
+        this.teamService = teamService;
```

Comment:
cia sito nereikia, pamirsai `Inject` padaryti TeamService, tada galesi getTeam metode pasiimti is klases scope, nebreikes i construktoriu padavineti